src/CLIMA.jl

```diff
@@ -1,6 +1,7 @@
 module CLIMA
 
 include(joinpath("Utilities", "TicToc", "TicToc.jl"))
+include(joinpath("Utilities", "ArtifactsWrappers", "ArtifactsWrappers.jl"))
```

Comment:
Only one niggling comment -- I understand why you used this name, but `ArtifactWrappers` is better English! ;)

Reply:
Updated!